discovery_engine_core/core/src/stack/ops/personalized.rs

```diff
@@ -47,6 +47,9 @@ impl Ops for PersonalizedNews {
     }
 
     async fn new_items(&self, key_phrases: &[KeyPhrase]) -> Result<Vec<Article>, GenericError> {
+        if key_phrases.is_empty() {
```

Comment:
Would this state occur when a new user starts using the app? Or is there another scenario? If it's the case of a new user, do we then fill the key phrases through the breaking news? 🤔

Reply:
It occurs for new users. A user interacts with the breaking news and we can start learning keywords. This can later happen also with active search.
I don't think we can go back to a state where we don't have keywords, even if we remove coi that are too old we want to keep a minimum in.